.github/CODEOWNERS

```diff
@@ -2,9 +2,7 @@
 # currently active dependency manager (DM) to trigger an automatic review
 # request from the DM upon changes. Please see AEP-002 for details:
 # https://github.com/aiidateam/AEP/tree/master/002_dependency_management
-setup.*                         @aiidateam/dependency-manager
```

Comment:
It looks like @aiidateam/dependency-manager doesn't exist? Should I assign @agoscinski explicitly?

Reply:
Done